src/hero_bringup/stationary_broadcaster.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
```

Comment:
This file is part of a project to automate object learning. After the evaluation we decided that it would be better to purchase a second camera to use for object learning, making this project obsolete.
I would suggest to remove this project entirely, or to move it to a separate branch.

Reply:
Keep it here till we actually switched.